src/snowflake/snowpark/functions.py

```diff
@@ -1651,6 +1651,27 @@ def to_decimal(e: ColumnOrName, precision: int, scale: int) -> Column:
     return builtin("to_decimal")(c, lit(precision), lit(scale))
 
 
+def to_double(e: ColumnOrName, fmt: Optional[ColumnOrLiteralStr] = None) -> Column:
```

Comment:
let's also add a test for this

Reply:
Added. I have not marked it as localtest though because the local test function does not have enough coverage to support the test yet.